packages/nextjs/scaffold.config.ts

```diff
@@ -1,9 +1,23 @@
+import { Abi } from "abitype";
 import * as chains from "wagmi/chains";
+import contracts from "~~/generated/hardhat_contracts";
```

Comment:
This still makes the app crash if you do `yarn start` and the file is not there.

How about:

```ts
let contracts;
try {
  contracts = require("~~/generated/hardhat_contracts");
} catch (error) {
  // Silently fail. The builder might not want to use a contract deployed from SE2
}
```

Reply:
I'm guessing that will break the narrow typing. I don't think there's a way to get around that. Next best thing we can do is explain very well in the setup that if the user wants a contract, he has to deploy it first, and if he doesn't want it, he should modify this in the config.

Reply:
> The aim of my PR is to make the changes a dev needs to make to use se-2 without contract as minimal as possible.

I have the same aim :)

What about?

```ts
// If you don't use custom contract for this SE-2 app:
// - Remove the import line below
// - Se "contracts: null" in the scaffoldConfig below
import contracts from "~~/generated/hardhat_contracts";

```

Reply:
Ah yes :+1: 

Would probably also be a good idea to update the readme a bit and explain how to use the scaffold.config

Reply:
db5d16bd28bdde35176faa7eb9da6dabef3549be

Reply:
I leave updating the README for another PR. We also need to include some other stuff.